support.md

```diff
@@ -40,38 +40,34 @@ The issue tracker (Redmine) should be used to check for existing issues or to fi
 
 When filing a new issue, check that the correct project is selected, then click the _New Issue_ link in the header.  Try to set the _Category_, _Found in release_ and include as much information as possible, including log files and screenshots.  For support requests, please use one of the other options below.
 
-## ![Google groups icon](static/images/google-groups.png) Mailing lists
+## ![Discourse icon](static/images/discourse.png) [Discourse Forum](https://community.theforeman.org)
 
-Mailing lists are available via Google Groups.  For help or support with
-Foreman problems, questions and answers or general discussion, post to the
-foreman-users list:
+We use a Discourse forum for our community interactions. For help or support
+with Foreman problems, questions and answers or general discussion, post to the
+Support board:
 
-* [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users) or email `foreman-users+subscribe@googlegroups.com`
+* [Support](https://community.theforeman.org/c/support) or email `users@community.theforeman.org` (account required)
 
-For discussions about developing Foreman or plugins, use the development list:
+For discussions about developing Foreman or plugins, use the Development board:
 
-* [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev) or email `foreman-dev+subscribe@googlegroups.com`
+* [Development](https://community.theforeman.org/c/development) or email `dev@community.theforeman.org` (account required)
 
-There is also a low-traffic list dedicated to release and security announcements:
+There is also a low-traffic board dedicated to release and security announcements:
 
-* [foreman-announce](https://groups.google.com/forum/?fromgroups#!forum/foreman-announce) or email `foreman-announce+subscribe@googlegroups.com`
+* [Release Announcements](https://community.theforeman.org/c/release-announcements) or subscribe via [RSS](https://community.theforeman.org/c/release-announcements.rss)
 
-### Mailing List Guidelines
+### Forum Guidelines
 
-In addition to the [general guidelines](/support.html#CommunityGuidelines) above, we also have a few email-specific guidelines:
+In addition to the [general guidelines](/support.html#CommunityGuidelines) above, we also have a few forum-specific guidelines:
 
 * Don't hijack threads - start new threads for new questions
-* Don't attach or paste in huge logs or attachments - upload them to a website
-  and include links in your post
-* Let us know if your question is answered - Replying to the thread with
-  *[SOLVED]* in the title lets others know that you no longer need help, and
-  helps others to find solutions when searching the archives
+* Don't attach or paste in huge attachments - upload them to a website and include links in your post
+* Let us know if your question is answered - the solution can be marked using
+  the thread tools, lets others know that you no longer need help, and helps
```

Comment:
maybe: `the thread tools. This lets others...`?

Reply:
Yeah, will fix on merge, ty